TaskHouseApi/Repositories/WorkerRepository.cs

```diff
@@ -0,0 +1,77 @@
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TaskHouseApi.DatabaseContext;
+using TaskHouseApi.Model;
+using TaskHouseApi.Repositories;
+
+namespace TaskHouseApi.Repositories
+{
+    public class WorkerRepository : IWorkerRepository
+    {
+    
+        private PostgresContext db;
+
+        public WorkerRepository(PostgresContext db){
+            this.db=db;
+        }
+
+        public async Task<Worker> Create(Worker w)
+        {
+            await db.Workers.AddAsync(w);
+            int affected = await db.SaveChangesAsync();
+
+            if (affected != 1)
```

Comment:
Formatting issue, please remember to look through your changes to catch these formatting issues before you commit and push to GitHub.
```suggestion
             if (affected != 1)
```

Reply:
Is there away you can see the whitespaces and enters as symbol?